src/frequenz/sdk/timeseries/logical_meter/_resampled_formula_builder.py

```diff
@@ -59,7 +59,7 @@ async def _get_resampled_receiver(self, component_id: int) -> Receiver[Sample]:
         return self._channel_registry.new_receiver(request.get_channel_name())
 
     async def push_component_metric(
-        self, component_id: int, nones_are_zeros: bool
+        self, component_id: int, *, nones_are_zeros: bool
```

Comment:
What * mean? It is not described.

Reply:
It is part of the Python syntax.  just means that all the arguments after the `*` are keyword-only i.e., they can only be passed with the name, like `push_component_metric(10, nones_are_zeros = False)`.

https://peps.python.org/pep-3102/